packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -47,11 +48,21 @@ import styles from './index.css'
   ),
 }))
 export default class HostItem extends Component {
+  state = {
+    assertConsistentHostServerTime: false,
+  }
+
   get _isRunning() {
     const host = this.props.item
     return host && host.power_state === 'Running'
   }
 
+  componentDidMount() {
+    assertConsistentHostServerTime(this.props.item).catch(() => {
+      this.setState({ assertConsistentHostServerTime: true })
```

Comment:
As i see, this property value is computed from props. So, it'll make sense to use a selector instead.

Reply:
But it on `componentDidMount`

Reply:
It's will be good if we do like this 
```js
const isHostServertTimeConsistent = createSelector(() => this.props.item.id, id => assertConsistentHostServerTime(id).then(() => true, () => false)
```
But it's not possible because it will return a promise